### DISCUSSION TITLE
Part Images - Optimal Sizing, Format and Reusing For Multiple Items

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#5258] [Q&A]
For the image that is provided for Parts. Is there an optimal size? The preview seems to be sized at 256x256, while the thumbnail is sized at 28x28. The Expanded version of the image opens a pop-up window with a 700x700 size.

I believe a 256x256 would be the recommended size, but unsure if there is an official recommendation - and I am leaning towards a JPG format, strictly for physical file size. 

For Parts that can use the same image (think screws, nuts, washers, etc.), is there a way to specify using an existing image already on the server without re-uploading a new image? In our case, for some materials, a generic image is sufficient. Currently, if we want to use the same image, we re-upload for each item, and the Inventree renames that duplicate create three versions (standard size as uploaded, thumbnail at 256x256, and preview at 128x128).

John

**Comment by SchrodingersGat** [comment]
This may be closely related to an existing issue - https://github.com/inventree/InvenTree/issues/3210

> For Parts that can use the same image (think screws, nuts, washers, etc.), is there a way to specify using an existing image already on the server without re-uploading a new image?

Yes, this can already be achieved: https://docs.inventree.org/en/latest/part/part/#web-interface
